app/UserService.ts

```diff
@@ -176,9 +203,35 @@ export class UserService extends ChordNode {
   // Inserts a User regardless of location in cluster
   async insert(message, callback) {
     const userEdit = message.request;
+    let isPrimaryHash: boolean = true;
```

Comment:
This feels a bit confusing to me. It looks sort of like error handling or something. Can you just directly put true and false as arguments to insertWithHash? We should be able to see what it means through intellisense.